modules/client/pom.xml

```diff
@@ -51,8 +51,20 @@
       <artifactId>jbossws-cxf-transports-undertow</artifactId>
       <version>${project.version}</version>
     </dependency>
-    
+
     <!-- CXF dependencies -->
+    <!-- https://mvnrepository.com/artifact/xerces/xercesImpl -->
+    <dependency>
+      <groupId>xerces</groupId>
+      <artifactId>xercesImpl</artifactId>
+      <version>2.7.1</version>
```

Comment:
xercesImpl version is managed, not needed to specify it here, see https://github.com/jbossws/jbossws-cxf/blob/master/pom.xml#L773

Reply:
Fixed. Thanks!